Dockerfile

```
@@ -1,32 +1,18 @@
 FROM ghcr.io/astral-sh/uv:python3.12-bookworm-slim
 
```

```suggestion
RUN uvx playwright install --with-deps chromium
```

I think specifying "chromium" leads to a smaller install? Maybe?